.gitignore

```diff
@@ -1,10 +1,6 @@
-# Ignore object files in this directory
-*.o
```

Comment:
I think these are not needed anymore, because they go into the `obj/` directory. Right?

Reply:
I believe you're right